docs/source/llms/openai/autologging.rst

```diff
@@ -4,6 +4,17 @@ MLflow OpenAI Autologging
 The OpenAI flavor for MLflow supports autologging to ensure that experimentation, testing, and validation of your ideas can be captured dynamically without 
 having to wrap your code with logging boilerplate. 
 
+.. attention::
+    Autologging is **only supported** for versions of the OpenAI SDK that are 1.17 and higher.
+
+MLflow autologging for the OpenAI SDK supports the following interfaces:
+
+- **Chat Completions** via ``client.chat.completions.create()``
+- **Completions** (legacy) via ``client.completions.create()``
+- **Embeddings** via ``client.embeddings.create()``
+
+Where ``client`` is either ``openai.OpenAI()`` or ``openai.AsyncOpenAI()``.
```

Comment:
client cannot be ``openai.AsyncOpenAI()``

Reply:
ahhh yep! fixing :)